backend/src/sms/models/sms-message-transactional.ts

```diff
@@ -55,4 +55,16 @@ export class SmsMessageTransactional extends Model<SmsMessageTransactional> {
 
   @Column({ type: DataType.STRING, allowNull: true })
   errorCode: string | null
+
+  @Column({ type: DataType.DATE, allowNull: true })
+  acceptedAt: Date
```

Comment:
if `allowNull` is `true`, the type here should be `Date | null` too

Reply:
missed that out, added